pyfritzhome/fritzhome.py

```diff
@@ -222,7 +222,7 @@ def get_alert_state(self, ain):
         device = self.get_device_by_ain(ain)
         return device.alert_state
 
-    def get_device_statistic(self, ain):
+    def get_device_statistics(self, ain):
```

Comment:
as you mentioned this will break the cli tool ... please change this too

Reply:
Should I revert the change?